running-a-nats-service/configuration/securing_nats/auth_intro/README.md

```diff
@@ -1,5 +1,9 @@
 # Authentication
 
+NATS authentication is multi-level. All the security mode have an [_accounts_](../../../../running-a-nats-service/configuration/securing_nats/auth_intro) level with [_users_](./#user-configuration-map) belonging to those accounts. The decentralized JWT Authentication also has an _operator_ to which the accounts belong. 
+
+Each account has its own independent subject name space: a message published on subject foo in one account not see by subscribers to foo in other accounts. Accounts can however define exports and imports of subject(s) streams as well as expose request-reply services between accounts. Users within an account will share the same subject name space but can be restricted to only be able to publish-subscribe to specific subjects.
```

Comment:
something is missing in this sentence - `a message published on subject foo in one account not see by subscribers to foo in other accounts`. Didn't know if the message should be seen or not seen.

Reply:
Thanks for spotting this. Fixed it in the new commit.